website/docs/r/key_vault_certificate_issuer.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+subcategory: "Key Vault"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_key_vault_certificate_issuer"
+description: |-
+  Manages a Key Vault Certificate Issuer.
+---
+
+# azurerm_key_vault_certificate_issuer
+
+Manages a Key Vault Certificate Issuer.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West US"
+}
+
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_key_vault" "example" {
+  name                = "example-key-vault"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku_name            = "standard"
+  tenant_id           = data.azurerm_client_config.current.tenant_id
+}
+
+resource "azurerm_key_vault_certificate_issuer" "example" {
+  name          = "example-issuer"
+  org_id        = "0000"
+  key_vault_id  = data.azurerm_key_vault.example.id
+  provider_name = "DigiCert"
+  account_id    = "0000"
+  password      = "example-password"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `account_id` - (Required) The account number with the third-party Certificate Issuer.
```

Comment:
This isn't actually required. I have a fork that implements this same resource provider and passing anything other than key_keyvault_id, provider_name, and name isn't required.

Reply:
@pwiens If your recommendation is to mark the attribute as `Optional` instead, then by all means.

Reply:
Yeah, that's what I would recommend. The reason is that we actually use it that way at my company. There is a pre-configured issuer and we just reference it. It's an internal issuer so not documented but yeah, we just pass the provider name and name. Making other attributes optional is the correct approach I think.

Reply:
Gotcha. So all but the `name` attribute should be marked `Optional` to be compatible with the way you use it now, is that correct?